### PR TITLE
tfd.Binomial does not need custom forward anymore

### PR DESCRIPTION
**Pull request by skeydan**
@junpenglao 

... and thanks for the help today (will probably be back ;-))

IDK if the notebook is in correct state, just erased output but some meta stuff will have sneaked in...

### CODE REVIEW

**Comment by skeydan** [comment]
> Check out this pull request on ReviewNB

--> probably this is not directed at me right?

**Comment by csuter** [comment]
> > Check out this pull request on ReviewNB
> 
> --> probably this is not directed at me right?

this is spam; ignore.

**Comment by junpenglao** [comment]
Thanks @skeydan! Could you only make the changes without removing the outputs?

**Comment by junpenglao** [comment]
The meta stuff changes makes it quite difficult to review - maybe the easiest way is to pull from master again and only edit the relevant  part in a text editor?

**Comment by skeydan** [comment]
but I think some cells get evaluated - should I evaluate those then? Or will that happen with the build process?

**Comment by junpenglao** [comment]
In this case I dont think you need to reevaluate those cells, as the custom function should gives the same format of output as the .sample()

**Comment by skeydan** [comment]
wow this was difficult, I hope I did it correctly :-)
(((in fact I _do_ appreciate R markdown even more now ;-)))

**Comment by junpenglao** [comment]
Yeah, plus the size of the file it might make editing pretty difficult - it is still showing a lot of changes in the diff tool, maybe you can try downloading the file from master and open it with a text editor? That way the states etc shouldnt be modified any more :-)

**Comment by skeydan** [comment]
oh but that's what I did? My `git diff` was pretty short, too...

The file from master had lots of binary, so I assume that was correct?

**Comment by skeydan** [comment]
also the diff here

https://github.com/tensorflow/probability/pull/382/files

is short...

File: tensorflow_probability/examples/jupyter_notebooks/Modeling_with_JointDistribution.ipynb [hunk 1]
```diff
@@ -2074,21 +2058,8 @@
...
-        "\n",
-        "  y = tfd.Binomial(\n",
-        "      total_count=tf.cast(at_bats, dtype),\n",
-        "      probs=thetas\n",
-        "  )\n",
-        "  return phi, kappa_log, thetas, y\n",
-        "\n",
-        "phi, kappa_log, thetas, y = custom_forward(4, n)\n",
-        "phi, kappa_log, thetas, y, y.log_prob(hits)"
+       "phi, kappa_log, thetas, y = mdl_baseball.sample(4)\n",
```
Comment (junpenglao) [inline tensorflow_probability/examples/jupyter_notebooks/Modeling_with_JointDistribution.ipynb]:
```suggestion
       "source": [
       "phi, kappa_log, thetas, y = mdl_baseball.sample(4)\n",
```

**Comment by junpenglao** [comment]
You are right! Left a comment.

**Comment by skeydan** [comment]
oops, thanks!

**Pull request merged**